src/components/FamilyManifestModal/queries.js

```diff
@@ -106,15 +106,16 @@ export const generateFamilyManifestModalProps = async ({ api, sqon }) => {
     participantIds,
     familyMembersWithoutParticipantIds,
   } = await familyMemberAndParticipantIds({ api, sqon });
-  const dataTypes = await dataTypesFromFamilyMemberIds({
-    api,
-    familyMemberIds: familyMembersWithoutParticipantIds,
-  });
   const { participantFilesCount, participantFilesSize } = await participantsFilesCountAndSize({
     api,
     sqon,
   });
+  const dataTypes = await dataTypesFromFamilyMemberIds({
+    api,
+    familyMemberIds: familyMembersWithoutParticipantIds,
+  });
   return {
+    loadingProps: false,
```

Comment:
what's this `loadingProps`? from what I can tell it is only used once and will always be false?

Reply:
it's `true` when the component loads and becomes `false` when `generateFamilyManifestModalProps` resolves.

Reply:
oh yeah I somehow missed that `generateFamilyManifestModalProps` was already in a life cycle hook :P yeah that'd make it easier to read definitely :)